scripts/ci/exports/utils/module.mjs

```diff
@@ -372,6 +377,15 @@ export class Module {
 		} );
 	}
 
+	_isFromPublicPackages( fileName ) {
+		// Checking if fileName contains the package that is marked as public API by default.
+		return PUBLIC_PACKAGES.some( publicPackageName => {
+			const publicPackageFullPath = upath.join( CKEDITOR5_ROOT_PATH, publicPackageName );
+
+			return fileName.includes( publicPackageFullPath );
+		} );
```

Comment:
When we prepare the target data format for the `PUBLIC PACKAGES` array, then we can simplify this util:

```suggestion
		return PUBLIC_PACKAGES.some( publicPackagePath => fileName.startsWith( publicPackagePath ) );
```